### DISCUSSION TITLE
Sidebar with sections and section title's

### DISCUSSION THREAD

**Discussion by onno-timmerman** [wagtail/wagtail#8295] [Ideas]
I think if you have a larger wagtail side the sidebar gets a bit unclear.
It would be nice if you can organise the wagtail admin sidebar in section. I would like to have like a CMS, MEDIA, EXTRA sections with nice dividers between them. This way the developer can organise and group all items in the sidebar. Currently (maybe I'm mistaken) you can only add items singel or grouped. 

Extra section you can name, or make more. For example I in my current project like to add a sections **Locations**, **productd**, **SEO stuff**. This is nice visual grouped way

**Comment by lb-** [comment]
@onno-timmerman thanks for the suggestion. 

Could you please add some more detail, we have a 'feature request template' that provides a great way to communicate features. 

https://github.com/wagtail/wagtail/blob/main/.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

For example it would be great to have a small mockup of what you are thinking (a screenshot with anotations), or how this may be implemented (maybe using the existing hooks system. 

The more details and context you can provide, the easier it is to give feedback and maybe for someone to pick this up and make it into a PR in the future. 

Thanks again for contributing this idea - I really like it.

**Comment by thibaudcolas** [comment]
Isn’t this exactly what https://docs.wagtail.org/en/stable/extending/admin_views.html#adding-a-group-of-menu-items makes possible already?

**Comment by onno-timmerman** [comment]
No see this mockup example:

When you use hooks a you could add a [ ] some list so you can set the where the menu comes to instead of that number?

<img width="408" alt="Untitled 2022-03-26 15-06-58" src="https://user-images.githubusercontent.com/178425/160243167-edc0fa8f-fdd1-4d09-b4c9-9e72a8814b5f.png">

**Reply by hazho** [reply]
if both of the menu and submenu have the same arrow, it may confuse the user, but your sections without the arrows or with +/- collapsible style would be great

**Comment by gasman** [comment]
Essentially this is an alternative design for what we currently achieve with submenus. I like the way it looks here, but there are definitely some tradeoffs - reducing the icon size means that there's a smaller target for mouse and touchscreen users, and I think it would need more work to arrive at a design that works equally well for users with short simple menus as well as 'power' users.

Given that we've only just completed a redesign of the sidebar, I think it's unlikely that we'll go for this any time soon, but I'll leave this open as a discussion.
